content/java/concepts/math-methods/terms/addExact/addExact.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: '.addExact()'
+Description: 'Returns the sum of its arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Function'
+  - 'Methods'
+  - 'Arithmetic'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`Math.addExact()`** method returns the sum of its arguments. It will throw an *exception* if the result overflows either int or long. This function is `static`, henceforth object creation is not required.
+
+## Syntax
+
+```pseudo
+Math.addExact(type a, type b)
+```
+Here `type` is the datatype of the argument passed to the function. **`Math.addExact()`** function accepts two arguments from the user and the datatype must be same for both the arguments. 
```

Comment:
```suggestion
The `Math.addExact()` method accepts two arguments from the user and the datatype must be same for both the arguments. 
```

Also, mention what types of augments it can accept.